packages/paste-website/src/components/customization-landing-page/ReadyToGetStarted.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import {Heading} from '@twilio-paste/heading';
+import {Column, Grid} from '@twilio-paste/grid';
+import {Card} from '@twilio-paste/card';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Anchor} from '@twilio-paste/anchor';
+
+import {LandingPageSection, LandingPageSectionContent} from './LandingPageLayoutUtils';
+
+export const ReadyToGetStarted: React.FC = () => {
+  return (
+    <LandingPageSection marginTop="space200" paddingTop={['space200', 'space200', 'space200']}>
+      <LandingPageSectionContent variant="default">
+        <Heading as="h2" variant="heading30">
+          Ready To Get Started?
+        </Heading>
+        <Grid
+          gutter={['space0', 'space40', 'space40', 'space40']}
+          vertical={[true, true, true, false]}
+          equalColumnHeights
+        >
+          <Column span={[12, 12, 12, 6]}>
+            <Grid gutter={['space40', 'space40', 'space40']} vertical={[true, false, false]} equalColumnHeights>
+              <Column span={[12, 6, 6]}>
+                <Card>
+                  <Heading as="h3" variant="heading30">
```

Comment:
```suggestion
                  <Heading as="h3" variant="heading40">
```

Reply:
For all of these cards, the heading size shouldn't be the same size as the main heading, I think heading 40 is the next size down, right?